src/JellyfishClientContext.tsx

```diff
@@ -122,7 +122,6 @@ const JellyfishContextProvider = (props: any) => {
     });
 
     await processIncomingMessages;
-    await membraneModule.create();
```

Comment:
why removing this?

Reply:
to fix issue when changing output device before meeting on android would not reflect those changes after connect,  right now there is a seperate function from react-native-membrane-webrtc `initializeWebRTC` that should be called only once in app, not every time when user connects to server

Reply:
is it documented that you have to use this function?

Reply:
in react-native-membrane-webrtc it is mentioned, also function itself is documented, but adding this to jellyfish repo would be a good idea

Reply:
also there are tutorials, I guess it should be also added there